functions/sendSystemMessage.ts

```diff
@@ -18,54 +17,78 @@ type EnvVars = {
 
 export type Body = {
```

Comment:
If you wanted to be super anal with your types you could do something like (assuming you agree with the comment about allowing both):

```
export type Body = ({
  taskSid?: string;
  channelSid: string} | {
  taskSid: string;
  channelSid?: string}) & {
  message?: string;
  from?: string;
  request: { cookies: {}; headers: {} };
};
```

But what you have is completely fine